packages/enzyme-adapter-react-helper/src/getAdapterForReactVersion.js

```diff
@@ -1,30 +1,40 @@
 import semver from 'semver';
 
+function isPreRelease(version) {
+  return semver.valid(version) && semver.SemVer(version).prerelease.length > 0;
+}
+
+function removePreRelease(version) {
+  return semver.inc(version, 'patch');
+}
+
 export default function getAdapterForReactVersion(reactVersion) {
-  const normalizedVersion = semver.coerce(reactVersion);
+  const versionRange = isPreRelease(reactVersion)
+    ? removePreRelease(reactVersion)
+    : semver.validRange(reactVersion);
```

Comment:
Great idea @ljharb about detecting prereleases and handling them as exact versions (which they are).